### ISSUE TITLE
Please add generation label in About Kiwi

### ISSUE THREAD

**Issue by rancidfrog** [kiwibrowser/src#230]
Currently only Version Git201030 shows in About Kiwi, which is not ideal, since releases are using `Generation 338511068`

![](https://i.imgur.com/WoFInzs.jpg) 

Some type of consistency is needed, or adding Generation version to about section.

Ideally, a notification of new version option in settings would be great. Several open source projects have implemented update notification which takes you to github release section on click.
- Newpipe ~ https://github.com/TeamNewPipe/NewPipe
- Fairemail ~ https://github.com/M66B/FairEmail

**Comment by kiwibrowser** [comment]
Very good idea, testing with https://github.com/kiwibrowser/src/commit/adc9fc36e517485ccb0cc8c9d82aa0f14db1c7a2

Ticket split @ #233

**Comment by rancidfrog** [comment]
Thank you for your quick response and commit
It works, but might need a space or "; " in between as seen below:
![](https://i.imgur.com/FAaHNvt.jpg) 

So,
it will be shown as,
`Git201105 Gen347353974` or `Git201105; Gen347353974``

**Comment by kiwibrowser** [comment]
Yes it's true, the version needs pretty-printing. I didn't do it, as technically it's safer to keep the version in one word (in case it's reused in settings or other places like to determine installation folder). I keep this issue open for now as you requested.

**Comment by kiwibrowser** [comment]
(until the version appears more beautifully)
